src/plugins/analysis/hash/view/hash.html

```diff
@@ -0,0 +1,14 @@
+{% extends "analysis_plugins/general_information.html" %}
+
+{% block analysis_result_details %}
+
+    {% for key, value in analysis_result.items() | sort %}
+        {% if value %}
+            <tr>
+                <td>{{ key }}</td>
+                <td style="font-family: monospace">{{ value }}</td>
+            </tr>
+        {% endif %}
+    {% endfor %}
```

Comment:
What do you think of explicitly listing all hashes here?
I think this improves readability.

Reply:
I don't like the idea: Not only is it a lot longer and very repetitive, it would also mean that we need to change the file each time we change the computed hashes